pep-0012.rst

```diff
@@ -137,13 +138,8 @@ directions below.
   prohibition of tab characters and the indentation requirements.
   See "Suggested Sections" below for a template of sections to include.
 
-- Update your References and Copyright section.  Usually you'll place
-  your PEP into the public domain, in which case just leave the
-  Copyright section alone.  Alternatively, you can use the `Open
-  Publication License`__, but public domain is still strongly
-  preferred.
-
-  __ http://www.opencontent.org/openpub/
```

Comment:
An option doesn't exist; all PEPs must be dual-licensed CC0 and "PD" (which is effectively the same as just licensing CC0, which in turn is a public domain declaration) per PEP 1 as implemented in #1117 per the discussion with PSF legal in #123 ; the lack of a corresponding update to PEP 12 was presumably an oversight. The OPL (which itself is an ambiguous acronym, as it could refer to either the Open Content License or the Open Publication License, two different licenses) is an obsolete, ambiguous, incompatible and likely legally flawed license; it would be strongly preferable to get those old PEPs relicensed, but that is entirely out of scope here.

Reply:
Ahh, thanks for the links -- important that this gets updated, then.

A